phpBB/styles/prosilver/theme/normalize.css

```diff
@@ -0,0 +1,331 @@
+/*! normalize.css v3.0.1 | MIT License | git.io/normalize */
+
+/**
+ * 1. Set default font family to sans-serif.
+ * 2. Prevent iOS text size adjust after orientation change, without disabling
+ *    user zoom.
+ */
+html {
+    font-family: sans-serif; /* 1 */
+    -ms-text-size-adjust: 100%; /* 2 */
+    -webkit-text-size-adjust: 100%; /* 2 */
+}
+
+/**
+ * Remove default margin.
+ */
+body { margin: 0 }
+
+/* HTML5 display definitions
+   ========================================================================== */
+/**
+ * Correct `block` display not defined for any HTML5 element in IE 8/9.
+ * Correct `block` display not defined for `details` or `summary` in IE 10/11 and Firefox.
+ * Correct `block` display not defined for `main` in IE 11.
+ */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+main,
+nav,
+section,
+summary { display: block }
+
+/**
+ * 1. Correct `inline-block` display not defined in IE 8/9.
+ * 2. Normalize vertical alignment of `progress` in Chrome, Firefox, and Opera.
+ */
+audio,
+canvas,
+progress,
+video {
+    display: inline-block; /* 1 */
+    vertical-align: baseline; /* 2 */
+}
+
+/**
+ * Prevent modern browsers from displaying `audio` without controls.
+ * Remove excess height in iOS 5 devices.
+ */
+audio:not([controls]) {
+    display: none;
+    height: 0;
+}
+
+/**
+ * Address `[hidden]` styling not present in IE 8/9/10.
+ * Hide the `template` element in IE 8/9/11, Safari, and Firefox < 22.
+ */
+[hidden],
+template { display: none }
+
+/* Links
+   ========================================================================== */
+/**
+ * Remove the gray background color from active links in IE 10.
+ */
+a { background: transparent }
+
+/**
+ * Improve readability when focused and also mouse hovered in all browsers.
+ */
+a:active,
+a:hover { outline: 0 }
+
+/* Text-level semantics
+   ========================================================================== */
+/**
+ * Address styling not present in IE 8/9/10/11, Safari, and Chrome.
+ */
+abbr[title] { border-bottom: 1px dotted }
```

Comment:
While it is recommended for usability/accessibility, I think it doesn't belong in prosilver.

Currently abbr tag is used only in footer (also one line in doc and debug output, but those abbreviations aren't something that users ever see, so they are irrelevant). Underline looks out of place there.

There are many other abbreviations in styles, but none of them use abbr tag so they are not affected: MCP, FAQ, PM, BBCode - those are used all over the forum. Unlike time zone, users see those abbreviations all the time. If abbreviations are changed, it should be done properly - all those abbreviations should also be changed to abbr tag.

Changing it only for one abbreviation that also appears to be the most irrelevant one is not the way to go.

As for reverting patch. Patch fixes an issue. Issue is abbreviations used to be displayed in prosilver without underline for ages and now suddenly it behaves differently in latest version of one browser. Patch fixes browser inconsistency, forcing Firefox to display abbreviations in same way as other browser do and old Firefix did. If you want to change layout of abbreviations, that's a different issue. That's what Igor fails to understand - that patch not about having or not having underline, its about fixing inconsistent layout caused by change in one browser. Adding underline is a different issue that needs to be discussed separately from that ticket (this ticket seems like a good place for that).